kubernetes/configs/prod/versions.sh

```diff
@@ -17,25 +17,25 @@ PURPOSE_PROCESS_IMAGE_VERSION="1.0.17"
 TENANT_MANAGEMENT_IMAGE_VERSION="1.0.10"
 TENANT_PROCESS_IMAGE_VERSION="1.0.11"
 API_GATEWAY_IMAGE_VERSION="1.0.18"
-BACKEND_FOR_FRONTEND_IMAGE_VERSION="1.0.17"
+BACKEND_FOR_FRONTEND_IMAGE_VERSION="1.0.18"
 NOTIFIER_IMAGE_VERSION="1.0.13"
 
 FRONTEND_IMAGE_VERSION="1.0.39"
 
 SELFCARE_ONBOARDING_CONSUMER_IMAGE_VERSION="1.0.8"
 
-JOB_ATTRIBUTES_LOADER_IMAGE_VERSION="1.0.21"
-JOB_DETAILS_PERSISTER_IMAGE_VERSION="1.0.21"
-JOB_TENANTS_CERTIFIED_ATTRIBUTES_UPDATER_IMAGE_VERSION="1.0.21"
-JOB_METRICS_REPORT_GENERATOR_IMAGE_VERSION="1.0.21"
-JOB_PADIGITALE_REPORT_GENERATOR_IMAGE_VERSION="1.0.21"
-JOB_DASHBOARD_METRICS_REPORT_GENERATOR_IMAGE_VERSION="1.0.21"
-JOB_ESERVICES_MONITORING_EXPORTER_IMAGE_VERSION="1.0.21"
-CERTIFIED_MAIL_SENDER_IMAGE_VERSION="1.0.21"
-ESERVICE_DESCRIPTORS_ARCHIVER_IMAGE_VERSION="1.0.21"
+JOB_ATTRIBUTES_LOADER_IMAGE_VERSION="1.0.22"
+JOB_DETAILS_PERSISTER_IMAGE_VERSION="1.0.22"
+JOB_TENANTS_CERTIFIED_ATTRIBUTES_UPDATER_IMAGE_VERSION="1.0.22"
+JOB_METRICS_REPORT_GENERATOR_IMAGE_VERSION="1.0.22"
+JOB_PADIGITALE_REPORT_GENERATOR_IMAGE_VERSION="1.0.22"
+JOB_DASHBOARD_METRICS_REPORT_GENERATOR_IMAGE_VERSION="1.0.22"
+JOB_ESERVICES_MONITORING_EXPORTER_IMAGE_VERSION="1.0.22"
+CERTIFIED_MAIL_SENDER_IMAGE_VERSION="1.0.22"
+ESERVICE_DESCRIPTORS_ARCHIVER_IMAGE_VERSION="1.0.22"
 
 JOB_DTD_CATALOG_EXPORTER_IMAGE_VERSION="1.0.8"
 JOB_ONE_TRUST_NOTICES_IMAGE_VERSION="1.0.8"
-JOB_ANAC_CERTIFIED_ATTRIBUTES_IMPORTER_IMAGE_VERSION="1.0.8"
+JOB_ANAC_CERTIFIED_ATTRIBUTES_IMPORTER_IMAGE_VERSION="1.0.10"
```

Comment:
This version is currently set in the test environment, I'm not sure it should be deployed in prod.
cc @galales

Reply:
we can safely deploy the last version also in production 👍